fmriprep/workflows/base.py

```diff
@@ -29,6 +32,11 @@ def base_workflow_enumerator(subject_list, task_id, settings):
         generated_workflow = base_workflow_generator(subject, task_id=task_id,
                                                      settings=settings)
         if generated_workflow:
+            generated_workflow.config['execution']['crashdump_dir'] =(
+                os.path.join(settings['output_dir'], 'log', subject)
```

Comment:
I see. Seems ok to me (although I still feel that a unique tempdir style folder would be more robust). I was trying to think of cases when the timestamp solution would break - when a user sends multiple jobs using the same output directory and the job happen to start at the same time. This is, however,  not a very likely scenario and many more things related to report generation would break in such case.